### DISCUSSION TITLE
DevTools failed to load SourceMap

### DISCUSSION THREAD

**Discussion by libyloba** [Chia-Network/chia-blockchain#4696] [Support]
hello, I was having a problem syncing and I decided to uninstall and reinstall the app now my when I add the plot directory it gives me this error:
![Screenshot (1)](https://user-images.githubusercontent.com/84055351/117991642-3ed41d00-b30c-11eb-862e-b4800ab640ff.png)
![Screenshot (2)](https://user-images.githubusercontent.com/84055351/117991713-501d2980-b30c-11eb-8003-8427d54a60c7.png)
![Screenshot (3)](https://user-images.githubusercontent.com/84055351/117991779-5c08eb80-b30c-11eb-8c73-56afe32402f9.png)

I also get these:
![Screenshot (4)](https://user-images.githubusercontent.com/84055351/117992029-9a060f80-b30c-11eb-924c-6fe995c9aac7.png)

and the syncing problem is still not fixed. 
![Screenshot (5)](https://user-images.githubusercontent.com/84055351/117992313-d5a0d980-b30c-11eb-9838-d89db920300d.png)
 
I have tried installing different versions but still the same errors. 
Thanks in advance, 
Erika.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
